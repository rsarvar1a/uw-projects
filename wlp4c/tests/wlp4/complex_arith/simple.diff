1a2
> 19
3c4
< $03 = 0x00000013
---
> $03 = 0x00000000
