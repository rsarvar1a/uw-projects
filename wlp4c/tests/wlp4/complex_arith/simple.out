Enter value for register 1: Enter value for register 2: Running MIPS program.
MIPS program completed normally.
$01 = 0x00000013   $02 = 0x0000002b   $03 = 0x00000013   $04 = 0x00000004   
$05 = 0x00000000   $06 = 0x00000000   $07 = 0x00000000   $08 = 0x00000008   
$09 = 0x00000000   $10 = 0x00000000   $11 = 0x00000001   $12 = 0x00000000   
$13 = 0x0000003e   $14 = 0x00000000   $15 = 0x0000002b   $16 = 0x00000000   
$17 = 0x0000003e   $18 = 0x00000000   $19 = 0x00000013   $20 = 0x00000000   
$21 = 0x0000003e   $22 = 0x00000000   $23 = 0x0000002b   $24 = 0x00000000   
$25 = 0x0000003e   $26 = 0x00000000   $27 = 0x00000000   $28 = 0x00000000   
$29 = 0x01000000   $30 = 0x01000000   $31 = 0x8123456c   
